src/api/workspace.rs

```diff
@@ -1158,6 +1159,25 @@ async fn put_workspace_default_published_view_handler(
   Ok(Json(AppResponse::Ok()))
 }
 
+async fn delete_workspace_default_published_view_handler(
+  user_uuid: UserUuid,
+  workspace_id: web::Path<Uuid>,
+  state: Data<AppState>,
+) -> Result<Json<AppResponse<()>>> {
+  let uid = state.user_cache.get_user_uid(&user_uuid).await?;
+  state
+    .workspace_access_control
+    .enforce_action(&uid, &workspace_id.to_string(), Action::Write)
```

Comment:
If only workspace owner should be allowed to delete default published view, you can use enfore_role instead.

Reply:
done